.env.example

```
@@ -0,0 +1,7 @@
+DATABASE_NAME=asl
+DATABASE_USERNAME=postgres
+DATABASE_PASSWORD=test-password
+DATABASE_HOST=postgres
+EMAILER_SERVICE=http://asl-emailer:8084
```

That will need to be `localhost`

```suggestion
EMAILER_SERVICE=http://localhost:8084
```

---

Good spot, exactly as I had it in my own .env file but forgot to update the example.